pyron-core/src/main/scala/com/cloudentity/pyron/domain/http/http.scala

```diff
@@ -85,7 +89,19 @@ object ApiResponse {
     ApiResponse(statusCode, body, headers)
 }
 
-case class OriginalRequest(method: HttpMethod, path: UriPath, queryParams: QueryParams, headers: Headers, bodyOpt: Option[Buffer], pathParams: PathParams)
+case class OriginalRequest(
+                            method: HttpMethod,
+                            path: UriPath,
+                            scheme: String,
+                            host: String,
+                            localHost: String,
+                            remoteHost: String,
+                            pathParams: PathParams,
+                            queryParams: QueryParams,
+                            headers: Headers,
+                            cookies: Map[String, String],
```

Comment:
So it would be pretty much exact same thing as PathParams, just named Cookie ... I think I didn't want to create yet another Cookie class, since there is already several in the libs and I think it could cause more confusion. To avoid reusing the same Cookie class name again, we'd have to name it something else but I can't come up with better name than Cookie to name a cookie :) PyronCookie ... in the end it's just a map and I came to the conclusion, it's maybe best to leave it as is. Let me know if you still want our own Cookie class, if so, what to name it.

Reply:
provided our own immutable Cookie

Reply:
provided our own immutable Cookie